pom.xml

```diff
@@ -711,9 +711,8 @@
 								com.google.cloud.firestore
 							</excludePackageNames>
 							<links>
-								<link>https://googleapis.dev/java/google-cloud-vision/latest/</link>
-								<link>https://googleapis.dev/java/google-cloud-spanner/latest/</link>
-								<link>https://googleapis.dev/java/google-cloud-clients/latest/</link>
+								<link>https://javadoc.io/doc/com.google.cloud/google-cloud-vision/latest/</link>
```

Comment:
I believe the official javadoc for client libraries that we point users to are the ones hosted on `https://c.g.c`?
like: `https://cloud.google.com/java/docs/reference/google-cloud-vision/latest/`

Reply:
I was experimenting with this locally but couldn't get it to work (pushed commit below) - I think this is because the plugin currently relies on and [looks for a `package-list` file](https://maven.apache.org/plugins/maven-javadoc-plugin/examples/links-configuration.html) after the `/` and is unable to find it.